src/Shopsys/ShopBundle/Resources/scripts/frontend/todo.js

```diff
@@ -21,4 +21,19 @@ $(document).ready(function () {
         $('js-filter-toggle-button').removeClass('active');
     });
 
+    // vzato z agáty, je potřeba nějak začlenit do rangeSlider.js .. funkčně to bere hodnoty z inputů, které máme skryté a dává je to do textových elementů, které zobrazuji. Po resetu filtrů by se měli aktualizovat, což se nyní neděje.
+    var showSliderPrices = function ( $slider ) {
+		var $minimalPriceInput = $ ( '#product_filter_form_minimalPrice' );
```

Comment:
it looks like extra spaces from somehow compliled javascript

Reply:
yes..but the file was only temporary and is completely removed in another commit :slightly_smiling_face: